build.gradle

```diff
@@ -26,14 +26,18 @@ buildscript {
     // For Apache Rat plugin to ignore non-Git files
     classpath "org.ajoberstar:grgit:1.5.0"
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.12.0'
+    classpath 'org.scoverage:gradle-scoverage:2.0.1'
   }
 }
 
 allprojects {
   apply plugin: 'idea'
   apply plugin: 'eclipse'
+  apply plugin: "jacoco"
+
   repositories {
     mavenCentral()
+    jcenter()
```

Comment:
This doesn't actually seem necessary based on the couple of targets I built? I'm not even sure what dependencies needed this? Same is true of the `jcenter()` reference at `buildscript` level, although that wasn't introduced in this patch.